models/src/main/scala/coop/rchain/models/rholang/RhoType.scala

```diff
@@ -143,8 +143,8 @@ object RhoType {
     def apply(bytes: Array[Byte]): Par = GDeployerId(bytes.toByteString)
   }
 
-  type RhoDeployId = DeployerId.type
```

Comment:
Is there really should be a `DeployerID` type here instead of `DeployID`?

Reply:
I think its a bug, fortunately `DeployerId.type` and `DeployId.type` are the same
It should be ` type RhoDeployId = RhoDeployId.type`